client/my-sites/exporter/post-type-options.jsx

```diff
@@ -0,0 +1,129 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import PureRenderMixin from 'react-pure-render/mixin';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import FormRadio from 'components/forms/form-radio';
+import Select from './select';
+import Label from 'components/forms/form-label';
+
+import { setPostType } from 'state/site-settings/exporter/actions';
+import {
+	getPostTypeOptions,
+	getSelectedPostType,
+} from 'state/site-settings/exporter/selectors';
+
+const mapStateToProps = ( state, ownProps ) => {
+	const siteId = state.ui.selectedSiteId;
+	const options = getPostTypeOptions( state, siteId, ownProps.postType );
+
+	return {
+		options,
+
+		// Show placeholders when options are not available
+		shouldShowPlaceholders: ! options,
+
+		// Disable options when this post type is not selected
+		isEnabled: getSelectedPostType( state ) === ownProps.postType,
+	};
+};
+
+const mapDispatchToProps = ( dispatch, ownProps ) => {
+	return {
+		onSelect: () => dispatch( setPostType( ownProps.postType ) )
+	};
+};
+
+/**
+ * Displays a list of select menus with a checkbox legend
+ *
+ * Displays a field group with a checkbox legend and optionally
+ * a list of select menus, or a description to appear beneath the
+ * legend.
+ */
+
+const PostTypeOptions = React.createClass( {
+	displayName: 'PostTypeOptions',
+
+	mixins: [ PureRenderMixin ],
+
+	propTypes: {
+		onSelect: PropTypes.func,
+
+		legend: PropTypes.string.isRequired,
+	},
+
+	renderPlaceholders() {
+		return (
+			<div className="exporter__option-fieldset-fields">
+				<div className="exporter__placeholder-text">
+					{ this.translate( 'Loading options…' ) }
+				</div>
+			</div>
+		);
+	},
+
+	renderFields() {
+		const { options } = this.props;
+
+		const Field = ( props ) => {
+			if ( ! props.options ) {
+				// This should be replaced with `return null` in React >= 0.15
```

Comment:
The current code will continue to work in React >= 0.15, it's just that `null` makes more sense than an empty `<span/>` once it's [supported in React 15](https://facebook.github.io/react/blog/#functional-components-can-now-return-null-too)